templates/grant_type/proposals.html

```diff
@@ -74,11 +74,7 @@ <h1>Proposals</h1>
     </li>
   </ul>
 
-  <p>Proposals are more likely to be approved if their relevance to the project is made clear, their scope is clearly defined, the proposer is qualified, and it's obvious that a lot of thought has gone into the idea.</p>
-
-  <a class="b bb" href="https://airtable.com/shrCi54rEDxgSZr3z" target="_blank">
-    Propose a project
-  </a>
+  <p>If you have an idea for how to improve Urbit, please read our <a class="bb" target="_blank" href="/grants/proposals/submit/">proposal submission guide</a>. Or, if you are ready, please feel free to <a class="bb" href="https://airtable.com/shrCi54rEDxgSZr3z" target="_blank"> propose a project</a>.
```

Comment:
Make sure the "propose a project" link remains bold and prominent, as it's the most important CTA. Probably best to say something like: 

Or, if you are ready, please feel free to:
**Propose a project**